src/battle_script_commands.c

```diff
@@ -7836,13 +7836,15 @@ static void BestowItem(u32 battlerAtk, u32 battlerDef)
 // Called by Cmd_removeitem. itemId represents the item that was removed, not being given.
 static bool32 TrySymbiosis(u32 battler, u32 itemId)
 {
+    u32 holdEffect;
```

Comment:
I don't think it matters here either, if anything the symbiosis check should go first.